app/models/dimensions/edition.rb

```diff
@@ -45,6 +45,7 @@ def metadata
     {
       title: title,
       base_path: base_path,
+      content_id: content_id,
```

Comment:
Could you please add a tests to it?
You would need to update:
1. https://github.com/alphagov/content-performance-manager/blob/master/spec/domain/queries/find_metadata_spec.rb
2. https://github.com/alphagov/content-performance-manager/blob/master/spec/requests/api/single_page_spec.rb